build_product

```diff
@@ -378,7 +378,7 @@ for chosen_product in "${_arg_product[@]}"; do
 	fi
 done
 
-CMAKE_OPTIONS=(${ADDITIONAL_CMAKE_OPTIONS} "${build_type_option}" "${oval_major_version_option}" "${oval_minor_version_option}" '-DSSG_PRODUCT_DEFAULT=OFF' "${cmake_enable_args[@]}" -G "$cmake_generator")
+CMAKE_OPTIONS=(${ADDITIONAL_CMAKE_OPTIONS} "${build_type_option}" "${oval_major_version_option}" "${oval_minor_version_option}" '-DSSG_PRODUCT_DEFAULT=OFF' '-DSSG_SCE_ENABLED:BOOL=ON' "${cmake_enable_args[@]}" -G "$cmake_generator")
```

Comment:
We have discussed this with the team and we think that we shouldn't enable SCE by default because at this moment we don't have any ways of ensuring quality of the contents of SCE files, ie. Automatus tests don't support SCE. Please remove this change. People can enable it on demand by adding the `'-DSSG_SCE_ENABLED:BOOL=ON` to the `ADDITIONAL_CMAKE_OPTIONS` bash variable.

I think that you have added this due to enablement of the tests in the CI. I think that instead this can be achieved by enabling it explicitly in the GitHub Actions definitions  in `.github/workflows/gate.yaml`. For example, it can be added to the Ubuntu job that builds the Ubuntu content because that already contains the SCE check. What do you think about this?

Reply:
Sure, I hadn't thought of using `ADDITIONAL_CMAKE_OPTIONS`. 
Would the new patch (that only enables it for Ubuntu 20.04 in the CI via the method your proposed) be acceptable in that regard?